src/ecs/commands/runCommandInContainer.ts

```diff
@@ -187,3 +146,58 @@ async function verifyCliAndPlugin(window: Window): Promise<boolean> {
     }
     return true
 }
+
+async function isDeploymentInProgress(node: EcsContainerNode): Promise<boolean> {
+    const deployments = (await node.ecs.describeServices(node.clusterArn, [node.serviceName]))[0].deployments
+    if (deployments) {
+        for (let i = 0; i < deployments.length; i++) {
```

Comment:
`for (const deployment of deployments)` and replace `deployments[i]` with `deployment`